### PR TITLE
Bring up to date

### PR DESCRIPTION
**Pull request by aterrype**
Upgrade Rails to 5.1 and fix deprecation issues

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ before_install:
   - tar -xzf geckodriver-v0.11.1-linux64.tar.gz -C geckodriver
   - export PATH=$PATH:$PWD/geckodriver
   - sudo service postgresql restart 9.3
+  - gem install bundler
```
Comment (wubr) [inline .travis.yml]: https://docs.travis-ci.com/user/languages/ruby/#bundler-20

File: Gemfile [hunk 2]
```diff
@@ -7,7 +7,7 @@ gem 'airbrake'          # robust exception tracking
 gem 'bootstrap-sass'    # SASS port of  Bootstrap 3
 gem 'devise'            # Flexible authentication solution
 gem 'devise_invitable'  # Invitation strategy for devise
-gem 'google_visualr'    # Google Visualization API
+gem 'google_visualr', git: 'https://github.com/winston/google_visualr' # Google Visualization API
```
[the remark this answers is not in the document]
Comment (aterrype) [reply Gemfile]: Yes.

File: config/environments/production.rb [hunk 3]
```diff
@@ -76,7 +76,7 @@
 
   # Enable locale fallbacks for I18n (makes lookups for any locale fall back to
   # the I18n.default_locale when a translation cannot be found).
-  config.i18n.fallbacks = true
+  config.i18n.fallbacks = [I18n.default_locale]
```
Comment (wubr) [inline config/environments/production.rb]: Did we end up needing this or was this left over from trying to figure out the UTF-8 parsing message?

Comment (aterrype) [reply config/environments/production.rb]: I changed that to fix a different deprecation issue after upgrading to Ruby 2.5. It's not related to the UTF-8 issue.

**Pull request merged**